druntime/test/aa/Makefile

```diff
@@ -14,4 +14,4 @@ $(ROOT)/%$(DOTEXE): $(SRC)/%.d
 	$(QUIET)$(DMD) $(DFLAGS) -of$@ $<
 
 clean:
-	rm -rf $(GENERATED)
+	rm -rf $(ROOT)
```

Comment:
the way this is spelled makes me nervous but I see
```
ROOT_OF_THEM_ALL = ../generated
ROOT = $(ROOT_OF_THEM_ALL)/$(OS)/$(BUILD)/$(MODEL)
```

Reply:
The ROOT and GENERATED in these integration-test Makefiles come from here: https://github.com/dlang/dmd/blob/97dc44aa769a7a252f7546bc446a8fd9a76fbc57/druntime/test/common.mak#L21-L22

They are local to each `druntime/test/<name>/` dir, while druntime's ROOT_OF_THEM_ALL is the `generated` dir at the repo top-level (so kills compiler too, which it shouldn't).